src/main/java/lesson2/Homework1.java

```diff
@@ -0,0 +1,22 @@
+package lesson2;
+
+import java.util.Scanner;
+
+public class Homework1 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Сколько лет вашей собаке сейчас?");
+        int x = in.nextInt();
+        System.out.println("Сколько месяцев вашей собаке сейчас?");
+        int r = in.nextInt();
+        System.out.println("Сколько лет вы хотите прибавить?");
+        int y = in.nextInt();
+        System.out.println("Сколько месяцев вы хотите прибавить?");
+        int u = in.nextInt();
+        int a, b;
+        a = x + y;
```

Comment:
I see you considered the amount of months and it is very good
for future - give please more meaningful names for your variables